4/Dockerfile

```
@@ -20,8 +20,9 @@ WORKDIR /opt
 RUN set -eux; \
     groupadd --system --gid 1000 scanner-cli; \
     useradd --system --uid 1000 --gid scanner-cli scanner-cli; \
+    curl -fsSL https://deb.nodesource.com/setup_lts.x | bash -; \
     apt-get update; \
-    apt-get install -y --no-install-recommends gnupg unzip wget bash fonts-dejavu python3 python3-pip shellcheck nodejs>12 build-essential; \
+    apt-get install -y --no-install-recommends gnupg unzip wget bash fonts-dejavu python3 python3-pip shellcheck nodejs build-essential; \
```

```suggestion
    apt-get install -y --no-install-recommends gnupg unzip wget bash fonts-dejavu python3 python3-pip shellcheck build-essential; \
```

I'm not sure node be installed via apt after running the nodesource script?

---

It should. The nodesource script just sets up the apt repository.